crates/ra_project_model/src/sysroot.rs

```diff
@@ -1,6 +1,7 @@
 use std::{
     path::{Path, PathBuf},
     process::Command,
+    vec::Vec
```

Comment:
unused import?

Reply:
from checking why `Vec` doesn't resolve :smile: I actually realized the problem now, the `extern crate` handling doesn't work correctly with 2015, I'll need to fix that as well...